### PR TITLE
Ensure that all configs are reloaded after reset + setup

### PR DESCRIPTION
**Pull request by smudge**
/domain @samandmoore @effron 
/platform @samandmoore @effron

### CODE REVIEW

**Comment by samandmoore** [comment]
I wonder if we can move the config loading after the check for enabled? It would be different from how things work right now but might be an improvement that avoids loading those files unless they need to be loaded? I can't think of anything that they'd need to be loaded for if setup is going to no op

**Comment by smudge** [comment]
> I wonder if we can move the config loading after the check for enabled? It would be different from how things work right now but might be an improvement that avoids loading those files unless they need to be loaded?

This makes sense -- I went ahead and made the change. Not seeing anything break locally when I point apps to it.

File: lib/webvalve/manager.rb [hunk 1]
```diff
@@ -149,6 +150,13 @@ def ensure_non_duplicate_stub(config)
       stubbed_urls << config.service_url
     end
 
+    def load_configs!
+      WebValve.config_paths.each do |root|
+        path = root.join('config', 'webvalve.rb').to_s
+        load path if File.exist?(path)
```
Comment (samandmoore) [inline lib/webvalve/manager.rb:156]: Don't love that we're doing ruby file io on every test case now, but we can look to optimize if it seems to actually be slow.

**Comment by samandmoore** [comment]
Okay. Let's do this!

<<domainlgtm platformlgtm

**Pull request merged**